Assets/MixedRealityToolkit/_Core/Interfaces/BoundarySystem/IMixedRealityBoundarySystem.cs

```diff
@@ -33,28 +33,54 @@ public interface IMixedRealityBoundarySystem : IMixedRealityEventSystem, IMixedR
         /// </summary>
         bool ShowFloor { get; set; }
 
+        /// <summary>
+        /// The physics layer that the generated floor is assigned to.
+        /// </summary>
+        int FloorPhysicsLayer { get; }
```

Comment:
```suggestion
        int FloorPhysicsLayer { get; set; }
```

Reply:
Adding setters doesn't do anything (Even in the prev version)

So are you saying you want me to add the logic to update the physics layers as well?

Reply:
> > We can add the setters later when it's actually implemented.
> 
> That would be another interface change. Let's go on the philosophy we have used in other areas and do all the forseen changes at once. I can help if needed.

Agreed. I'm not sure if the MRTK team has a philosophy on read-only properties specifically but in .Net in general they're very rare. And when they do exist it is for a specific reason (e.g. it wouldn't make sense for code to change the `.Position` property of a GPS receiver). 

I understand the expectation is that these values will be set in the Editor. It appears the expectation is that they'll never need to be changed at runtime. If that's the case, there is very little value in exposing them as properties at all. However, if we agree that there may be a need to change these values at runtime (and I already have an existing case where they would need to be) then having the properties as read-only just adds frustration for developers.

I would like to suggest that we be very deliberate in our choices to implement read-only properties anywhere. In this specific case, if these properties are exposed I do feel they should be read/write. And yes I understand that that would mean implementing logic to update the layer if the prefab has already been instantiated at the time the property is set.

Reply:
I think our coding guidelines state to make things private by default and only expose them when needed.

Reply:
In either case, I updated the PR with the setters that also update the generated objects.